homeassistant/components/vesync/fan.py

```diff
@@ -22,7 +22,7 @@
 }
 
 SPEED_AUTO = "auto"
-FAN_SPEEDS = [SPEED_AUTO, SPEED_OFF, SPEED_LOW, SPEED_MEDIUM, SPEED_HIGH]
+FAN_SPEEDS = [SPEED_OFF, SPEED_LOW, SPEED_MEDIUM, SPEED_HIGH, SPEED_AUTO]
```

Comment:
`auto` is not an allowed fan speed. It should be removed as I asked in the follow up comment on your PR for this integration.

Reply:
In the previous one it was agreed to keep it in until the other pull request was completed because removing it breaks an important feature of the integration. This just corrects the order so it at least sets the right speed. I understand it is not standard and can remove it, but I thought it was ok for now until the new approach was ready.

Reply:
No, it's not ok to keep it.

https://github.com/home-assistant/core/pull/36132#discussion_r483404837

Reply:
Sorry I honestly never saw your last comment or else I would have removed it. Pushed a commit to remove it from the speed list instead.